model/main.py

```diff
@@ -0,0 +1,27 @@
+from model.model import Model
+
+from argparse import ArgumentParser, Namespace
+
+
+def get_args() -> Namespace:
```

Comment:
I think it's better to split this into two separate components - setting up the parser, and parsing the arguments. This method should only setup the parser, and another method/function can make use of it to parse the arguments.